### ISSUE TITLE
Streaming not working when using differents wifi networks

### ISSUE THREAD

**Issue by miguelwon** [lanikai/alohacam#5]
Everything works fine when I'm connected to the same network (browser in laptop and camera in the raspberry pi). 

But I tried the following: left `alohacam` running in the background and then switched the laptop wifi to an external one (used my mobile 4G as hotspot). With this setup, the stream does not work. After connecting, it only shows the first frame and then it gets frozen.

If I switch back the laptop wifi to the local network and connect, it starts to work ok.

I'm using Rasp 4 with:

```
No LSB modules are available.
Distributor ID:	Raspbian
Description:	Raspbian GNU/Linux 10 (buster)
Release:	10
Codename:	buster
```

**Comment by hiszpanski** [comment]
The available bandwidth over 4G connection may be insufficient (https://www.speedtest.net can give you an idea of available bandwidth). In v0.1.6, the video bitrate is static, but you can adjust it using the `--bitrate` flag (try `--bitrate=1000000` for 1Mbps video). Adaptive bitrate support (i.e. congestion control) is in development and is slated for release in v0.1.7.

**Comment by miguelwon** [comment]
Ok, thanks. 
Yes, it was related to the bitrate. I set to 250kbps just to make sure and it worked fine. 
I measured a latency time of about 230ms. Very nice from what I have seen!
Any plans to make source code public?

**Comment by hiszpanski** [comment]
Thanks! Happy to hear it's working for you.

At the moment, we haven't been able to find a business model that would let us open-source the code, although if that changes in the future we would be happy to do so as we share the belief that open source code makes software more valuable. But we also need to feed our families.

If you are worried that we're eavesdropping on the video or metrics, we're not -- feel free to use Wireshark to inspect the network traffic. We make Alohacam available for free to show developers what is possible, and get real-world testing on our code base.

**Comment by miguelwon** [comment]
Hi again,
Ok, thanks. Do you have any plans to offer it as a service? I'm working on
a project (seed like idea and trying to build a proof-of-concept) that
would require WebRTC. I'm no expert in this and so one possibility would be
to use part of your solution. "Part" because I just need the script located
in the raspberry pi. I would just need the websocket address for the
signaling as a parameter (I think).
Regards,
Miguel

Chris Hiszpanski <notifications@github.com> escreveu no dia quinta,
31/10/2019 à(s) 21:00:

**Comment by hiszpanski** [comment]
Yes, we'll be launching a service next month. Email us at aloha@lanikailabs.com and will make sure you're notified when this happens.
